go/plugins/googlegenai/gemini.go

```diff
@@ -79,6 +79,29 @@ type EmbedOptions struct {
 	TaskType string `json:"task_type,omitempty"`
 }
 
+type GeminiEmbeddingConfigSchema struct {
```

Comment:
We no longer want to define our own config types, we should try to use them directly from the gen-ai SDK.